src/Workspaces/Core/Portable/FindSymbols/DeclaredSymbolInfo.cs

```diff
@@ -72,12 +94,35 @@ internal static DeclaredSymbolInfo ReadFrom(ObjectReader reader)
                 var parameterCount = reader.ReadUInt16();
                 var typeParameterCount = reader.ReadUInt16();
 
-                return new DeclaredSymbolInfo(name, immediateContainer, entireContainer, kind, new TextSpan(spanStart, spanLength), parameterCount, typeParameterCount);
+                var inheritanceNamesLength = reader.ReadInt32();
```

Comment:
Do we have version numbers on the caches that this writes into/reads from so that we don't break people who upgrade?